pkg/handlers/default.go

```diff
@@ -49,3 +56,68 @@ func KillContainer(pid int) registry.HandlerFunc {
 		return registry.HandlerResultErrno(unix.EPERM)
 	}
 }
+
+func FreezeContainer(pid int) registry.HandlerFunc {
+	re := regexp.MustCompile(`^(\d*):(freezer)?:(/..*)$`)
```

Comment:
Why outside the function we return in the next line?

Reply:
Just because there is no need to re-compile the regexp for each syscall. We can compile it one time when the container is configured with its seccomp policy.

Reply:
I moved it outside of the function, with comments.